examples/training/object_detection/pascal_voc/retina_net.py

```diff
@@ -38,9 +60,38 @@
 ]
 class_mapping = dict(zip(range(len(class_ids)), class_ids))
 
+flags.DEFINE_string(
+    "weights_path",
+    "weights_{epoch:02d}.h5",
+    "Directory which will be used to store weight checkpoints.",
+)
+flags.DEFINE_string(
+    "tensorboard_path",
+    "logs",
+    "Directory which will be used to store tensorboard logs.",
+)
+FLAGS = flags.FLAGS
+FLAGS(sys.argv)
+
+# parameters from RetinaNet [paper](https://arxiv.org/abs/1708.02002)
```

Comment:
What's the "parameters" here?

Reply:
Everything from batch size, learning rate, to image size, etc.